packs/elastic/scripts/logsfeed.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
```

Comment:
Ok I'll move them, docker-compose.yml as well?

It's not a handy convention for `docker-compose.yaml`. Since if it's in 
the root of `elasticsearch/`it will be nice to build, run and exec 
because containers get names like **elasticsearch_st2_1** and 
**elasticsearch_elk_1**.

For scripts it would be nice, but for `docker-compose.yaml` I would 
prefer to leave it in its place.

On Tue, May 19, 2015 at 2:17 PM, Tomaz Muraus 
<notifications@github.com> wrote:
> In packs/elastic/scripts/logsfeed.py:
> 
> > @@ -0,0 +1,99 @@
> > +#!/usr/bin/env python
> Comments from the previous PR still apply - please move additional 
> scripts and files into etc/ directory.
> 
> —
> Reply to this email directly or view it on GitHub.
>